udl-list.md

```diff
@@ -1,7 +1,265 @@
+| [RTF Rich Text Format](./UDLs/RichText-RTF_byFrankRalf.xml) | RTF Rich Text Format | [Frank Ralf](https://github.com/nakohdo/NPP.RTF) |
+| [Rebol](./UDLs/Rebol_byMasaruYamabashi.xml) | Rebol | Masaru Yamabashi |
+| [RenderMan Shading Language &amp; RIB](./UDLs/RenderMan-RSL_byStefanGustavson.xml) | RenderMan Shading Language &amp; RIB | Stefan Gustavson |
+| [RenderMan Shading Language v2 autocompetion file](./UDLs/RenderMan-RSL_by-focus_gfx.xml) | RenderMan Shading Language v2 autocompetion file | focus_gfx |
+| [Rules Language](./UDLs/Rules_Language_byNimadDeshpande.xml) | Rules Language | Nimad Deshpande |
+| [SAP NetWeaver IdM language](./UDLs/SAP-NetWeaver-IdM-v7p1_byFrankBuchholz.xml) | SAP NetWeaver IdM language | Frank Buchholz |
+| [SAP Web Intelligence Formula Editor](./UDLs/SAP-WebIntelligenceFormula_byIvanRubil.xml) | SAP Web Intelligence Formula Editor | Ivan Rubil |
+| [SAS with pseudocode](./UDLs/SAS_withPseudocode_byJeffFoarde.xml) | SAS with pseudocode | Jeff Foarde |
+| [SAS](./UDLs/SAS_byCathLawrence.xml) | SAS | Cath Lawrence |
+| [SLAX](./UDLs/SLAX_byTravisWaltz.xml) | SLAX | Travis Waltz |
+| [SPARQL 1.1](./UDLs/SPARQL-v1p1_byFranckMichel.xml) | SPARQL 1.1 | Franck Michel |
+| [SPlunk configuration](./UDLs/SplunkConfig_by-httpstergeek.xml) | SPlunk configuration | httpstergeek |
+| [SQR](./UDLs/SQR_byJames.xml) | SQR | James |
+| [STAPL](./UDLs/STAPL_byMichaelWalker.xml) | STAPL | Michael Walker |
 | [STL 3dObject](./UDLs/STL-3dObject-ASCII.byPryrt.xml) | 3d Stereolithography ASCII file (STL) | [Pryrt](https://github.com/pryrt/)|
+| [STOL](./UDLs/STOL_byPeterNell.xml) | STOL | Peter Nell |
+| [Samcef](./UDLs/Samcef_by-moleculext.xml) | Samcef | [François](https://github.com/moleculext/npp_samcef) |
+| [Sap ABAP](./UDLs/SAP-ABAP_byFrancoCapetta.xml) | Sap ABAP | Franco Capetta |
+| [Schlumberger Eclipse](./UDLs/SchlumbergerEclipse-slb_by-juanbretti.xml) | Schlumberger Eclipse | Juan Pedro Bretti Mandarano |
+| [SciLab (highlighting and autocompletion)](./UDLs/SciLab_bySamuelGougeon.xml) | SciLab (highlighting and autocompletion) | [Samuel Gougeon](http://fileexchange.scilab.org/toolboxes/140000) |
+| [ShadeLab](./UDLs/ShadeLab_by-redcool007.xml) | ShadeLab | redcool007 |
+| [Siebel eScript](./UDLs/Siebel_eScript_byGiuseppeGarofalo.xml) | Siebel eScript | Giuseppe Garofalo |
+| [Siemens SCL (definition and autocompletion)](./UDLs/Siemens-SCL_byTizianoSartini.xml) | Siemens SCL (definition and autocompletion) | Tiziano Sartini |
+| [Skyrim Papyrus language](./UDLs/Skyrim-Papyrus_byJohnKostrzewski.xml) | Skyrim Papyrus language | John Kostrzewski |
+| [SmallWorld Magik](./UDLs/SmallWorldMagik_byChristianTaton.xml) | SmallWorld Magik | Christian Taton |
+| [Smarty](./UDLs/Smarty_byHelge-deVries.xml) | Smarty | Helge de Vries |
+| [Snort](./UDLs/Snort_by-tropismgroup.xml) | Snort | http://www.tropismgroup.org/2012/08/02/snort-user-defined-language-udl-in-notepad/ |
```

Comment:
The content of URL is not readable.
Let's put Anonymous, and remove the URL for the moment.

Reply:
URL removed.  Since I had called it `Snort_by-tropismgroup`, I decided to use same identifier as the author.
https://github.com/pryrt/userDefinedLanguages/commit/6f8f59e85f21add32590c3d3f318852156db7cc5

Reply:
Item above

Reply:
https://github.com/pryrt/userDefinedLanguages/commit/e70fd6cce6bb35581fffb58806560432af86475d switched to anonymous